src/test/docbook/pass/enumsynopsis.001.xml

```diff
@@ -54,7 +54,11 @@
             </db:enumitem>
             <db:enumitem>
                <db:enumidentifier>AlignHorizontal_Mask</db:enumidentifier>
-               <db:enumvalue>AlignLeft | AlignRight | AlignHCenter | AlignJustify | AlignAbsolute</db:enumvalue>
+               <db:enumvalue>AlignLeft</db:enumvalue>
```

Comment:
I'm assuming that the semantic here is `AlignHorizontal_Mask` is an enumeration and that the value it has must be exactly one of `AlignLeft`, `AlignRight`, etc. Rendering that as `AlignLeft | AlignRight |...` makes perfect sense. But the individual enumeration values aren't very useful stuck in a big text field.

I presume, for example, that one might to refer to the `AlignLeft` enumeration value in prose somewhere and might want to link to it. Or perhaps build a specialized index of enumerations, with entries for each value. In either case, you'd want the values identified discretely, not in a long field that has to be microparsed.

Perhaps the markup is insufficient for that task, I'm not really sure, but structured values that have to be microparsed seem like a bad design.

Reply:
There is no need for microparsing here: using `AlignHorizontal_Mask` is like using all `AlignLeft`, `AlignRight`, etc. at the same time. It's a very useful shortcut to avoid typing many other enumeration values. All these values are defined elsewhere in the same enumeration, it's just a supplementary value that is useful in practice (but it could very well be omitted without losing functionality, just convenience). 

In your case of an index of enumeration values, you would have these entries (and the part that should be linked in the index): [`AlignLeft`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L27), [`AlignRight`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L35), [`AlignHCenter`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L43), [`AlignJustify`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L47), [`AlignAbsolute`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L51), [`AlignHorizontal_Mask`](https://github.com/ndw/docbook/blob/0ca0b4608d84e604ba11b193633b37af01091f39/src/test/docbook/pass/enumsynopsis.001.xml#L55).